extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -244,46 +231,13 @@ export const BackupPhraseWarningMessage = () => {
 };
 
 interface BlockaidAssetWarningProps {
-  code?: string;
-  issuer?: string;
-  isNewAsset?: boolean;
   blockaidData: BlockAidScanAssetResult;
 }
 
 export const BlockaidAssetWarning = ({
-  isNewAsset,
-  code,
-  issuer,
   blockaidData,
 }: BlockaidAssetWarningProps) => {
   const { t } = useTranslation();
-  const networkDetails = useSelector(settingsNetworkDetailsSelector);
-  const [isNewAssetState, setIsNewAssetState] = useState(false);
-
-  useEffect(() => {
-    const fetchSuspiciousAsset = async (
-      assetCode: string,
-      assetIssuer: string,
-    ) => {
-      const server = stellarSdkServer(
-        networkDetails.networkUrl,
-        networkDetails.networkPassphrase,
-      );
-      const resp = await checkForSuspiciousAsset({
-        code: assetCode,
-        issuer: assetIssuer,
-        domain: "",
-        server,
-        networkDetails,
-      });
-
-      setIsNewAssetState(resp.isNewAsset);
```

Comment:
we don't need to manually check for new assets with stellar.expert - blockaid handles this now